package.json

```diff
@@ -13,17 +13,16 @@
 		"@types/react": "^17.0.19",
 		"@types/react-dom": "^17.0.9",
 		"@types/styled-components": "^5.1.13",
+		"eslint": "^7.32.0",
+		"lint-staged": "^11.1.2",
+		"prettier": "^2.4.0",
 		"react": "^17.0.2",
 		"react-dom": "^17.0.2",
 		"react-query": "^3.21.1",
-		"react-scripts": "4.0.3",
+		"react-scripts": "^4.0.3",
 		"styled-components": "^5.3.1",
 		"typescript": "^4.4.2",
-		"web-vitals": "^1.1.2",
-		"eslint": "^7.32.0",
-		"husky": "^7.0.2",
-		"lint-staged": "^11.1.2",
-		"prettier": "^2.4.0"
+		"web-vitals": "^1.1.2"
```

Comment:
Why web vitals ??

Reply:
I didn't install that. dunno 😂 but I did run `npm update`